src/evict/evict_inline.h

```diff
@@ -453,8 +622,21 @@ __wti_evict_hs_dirty(WT_SESSION_IMPL *session)
 }
 
 /*
+ * NO_AUTO_FORMAT
+ *
  * __wt_evict_app_assist_worker_check --
- *     Evict pages if the cache crosses eviction trigger thresholds.
+ *     This function is called by application threads to check whether eviction trigger thresholds
+ *     are reached and assist the eviction worker threads with eviction of pages from the queues.
```

Comment:
```suggestion
 *     are reached and the thread should assist eviction worker threads with eviction of pages from the queues.
```

Reply:
Updated in [73744c3](https://github.com/wiredtiger/wiredtiger/pull/11096/commits/73744c3a2b3e6a9811054165f507c2971025785e)